nnpdf_data/nnpdf_data/new_commondata/STAR-2006_SHP_200GEV/metadata.yaml

```diff
@@ -0,0 +1,51 @@
+# Generalia
+setname: "STAR-2006_SHP_200GEV"
+
+version: 1
+version_comment: "Initial implementation"
+
+# References
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1253360"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1253360"
+  version: 1
+
+nnpdf_metadata:
+  nnpdf31_process: "SHP" # Single Hadron Production
+  experiment: "STAR"
+
+implemented_observables:
+  - observable_name: "ALL"
+    observable:
+      description: "Neutral pion spin asymmetries at intermediate pseudorapidity in polarized proton collisions sqrt(s)=200 GeV"
+      label: "$A_{LL$"
+      units: ""
+    process_type: "SHP_ASY"
```

Comment:
please add a process to the process option library

Reply:
Yes, yes. I was planning to do so but then more things came up and pushed this down to the list of priorities ^^U